container/docker/handler.go

```diff
@@ -312,7 +314,7 @@ func (self *dockerContainerHandler) ListContainers(listType container.ListType)
 		}
 
 		ref := info.ContainerReference{
-			Name:      FullContainerName(c.ID),
+			Name:      FullContainerName(c.ID, self.cgroupParent),
```

Comment:
This is actually redundant as this is only used for non-systemd Dockers where `self.cgroupParent` is empty.